pkg/apis/config/doc.go

```diff
@@ -0,0 +1,16 @@
+/*
+Copyright 2020 The Kubernetes Authors.
```

Comment:
can we add new lines so the license header looks like this:
```
/*
Copyright 2020 The Kubernetes Authors.

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.

You may obtain a copy of the License at
    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*/
```

This is also consistent with the rest of the repo

Reply:
gotcha, will update!

Reply:
There were some other files that needed this chagne as well

Reply:
yeah, I've also updated them :)